crates/matrix-sdk/src/encryption/mod.rs

```diff
@@ -230,8 +230,9 @@ impl Client {
         Ok(self.send(request, None).await?)
     }
 
+    /// Create a direct message room with the specified user.
     #[cfg(feature = "e2e-encryption")]
-    pub(crate) async fn create_dm_room(
+    pub async fn create_dm_room(
         &self,
         user_id: OwnedUserId,
```

Comment:
I think we should make this `&UserId` for consistency with other (public) methods. I think the only change required for this internally is changing `user_id.clone()` to `user_id.to_owned()`.

Reply:
Done